### DISCUSSION TITLE
"Group expectations"

### DISCUSSION THREAD

**Discussion by glaucocustodio** [evenbetterspecs/evenbetterspecs.github.io#8] [General]
What are your thoughts on https://evenbetterspecs.github.io/#group-expectations?

**Comment by unikitty37** [comment]
I like to add the `:aggregate_failures` flag to tests with multiple expectations; without this, RSpec will only show the first failure in a test, which can make it harder to work out _why_ it's failing — or sometimes even leads to the wrong fix being implemented initially.

**Reply by glaucocustodio** [reply]
Thank you so much @unikitty37 for letting us know about this option. 

Perhaps we can suggest the use of such a feature on this guideline via `define_derived_metadata` as per https://rspec.toolboxforweb.xyz/docs/rspec-core/expectation_framework_integration/aggregating_failures#enable-failure-aggregation-globally-using-define_derived_metadata. 

Would like to open a PR for that?

**Reply by glaucocustodio** [reply]
I just did it.
